lib/api-request.js

```diff
@@ -129,7 +129,7 @@ module.exports = function (api_url, public_key, private_key) {
 
   */
 
-    var lowercase       = lowercase || true
```

Comment:
This always evaluates to true so even passing lowercase = false in the arguments didn't work.